ScreenRuler/CalibrationForm.resx

```diff
@@ -193,16 +193,16 @@
     <value>NoControl</value>
   </data>
   <data name="label11.Location" type="System.Drawing.Point, System.Drawing">
-    <value>33, 47</value>
+    <value>33, 43</value>
   </data>
   <data name="label11.Size" type="System.Drawing.Size, System.Drawing">
-    <value>96, 13</value>
+    <value>101, 12</value>
   </data>
   <data name="label11.TabIndex" type="System.Int32, mscorlib">
     <value>9</value>
   </data>
   <data name="label11.Text" xml:space="preserve">
-    <value>Display Scaling (%)</value>
+    <value>显示缩放比例 (%)</value>
```

Comment:
Please add the translation in a separate file (`CalibrationForm.zh.resx` ?) as otherwise the English translation will be overriden.